### DISCUSSION TITLE
可否设置鼠标双击任务栏小图标截图识别文字

### DISCUSSION THREAD

**Discussion by sxxstc** [hiroi-sora/Umi-OCR#349] [Ideas]
如题

**Comment by hiroi-sora** [comment]
技术上没有难度，主要是写 设置这个功能的UI面板有点麻烦。

你可以自己改代码来增加这个功能用着先：

用记事本打开 `UmiOCR-data\qt_res\qml\MainWindow\SystemTray.qml`

拉到最后，有一坨 `onActivated:` 的代码块，改成：

```
    onActivated: {
        if(reason == SystemTrayIcon.DoubleClick) // 双击
            qmlapp.mainWin.setVisibility(true) // 主窗可见
        if(reason == SystemTrayIcon.Trigger) // 单击
            qmlapp.pubSub.publish("<<screenshot>>") // 发送截图事件
    }
```

这样，单击图标是截图，双击是弹出主窗口。你也可以将`DoubleClick`与`Trigger`反过来。

注意，必须先打开截图识别的标签页，才能正确的调用截图。
